content/batch/workflow-simple.md

```diff
@@ -47,11 +47,12 @@ Duration:            3 seconds
 STEP               PODNAME         DURATION  MESSAGE
  ✔ whalesay-2kfxb  whalesay-2kfxb  2s        
 ```
+Make a note of the workflow's name from your output.
 
-Confirm the output.
+Confirm the output by running the following command:
```

Comment:
If we sub out the workflow name, then we should also state to substitute the name out...

“Confirm the output by running the following command, substituting `<workflow name>` for the name of the workflow above.”

Reply:
Yes you are right. My text is actually:
"Make a note of the workflow's name from your output.

Confirm the output by running the following command:

```bash
argo logs <workflow name>
```"
But your sentence is much clearer. Will make changes and commit

Reply:
looks like this is fixed